src/main/java/me/iblitzkriegi/vixio/expressions/retrieve/ExprUserWithId.java

```diff
@@ -6,37 +6,31 @@
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 import me.iblitzkriegi.vixio.Vixio;
-import net.dv8tion.jda.core.entities.Guild;
-import net.dv8tion.jda.core.entities.Member;
+import me.iblitzkriegi.vixio.util.Util;
+import me.iblitzkriegi.vixio.util.wrapper.Bot;
 import net.dv8tion.jda.core.entities.User;
 import org.bukkit.event.Event;
 
 public class ExprUserWithId extends SimpleExpression<User> {
     static {
         Vixio.getInstance().registerExpression(ExprUserWithId.class, User.class, ExpressionType.SIMPLE,
-                "user with id %string% [in %guild%]")
+                "user with id %string%")
                 .setName("User with ID")
                 .setDesc("Get a User via their ID.")
-                .setExample("name of user with id \"456145141891891\"");
+                .setExample("broadcast name of user with id \"456145141891891\"");
     }
 
     private Expression<String> id;
-    private Expression<Guild> guild;
 
     @Override
     protected User[] get(Event e) {
-        Guild guild = this.guild.getSingle(e);
         String id = this.id.getSingle(e);
-        if (guild == null || id == null) {
+        Bot bot = Util.randomBot();
```

Comment:
You should check all the bots until one can find it, not just pick random one and just try.